pkg/ccl/backupccl/restore_job.go

```diff
@@ -1179,6 +1179,7 @@ func createImportingDescriptors(
 		for _, desc := range mutableTables {
 			if desc.GetParentID() == tempSystemDBID {
 				desc.SetPublic()
+				desc.LocalityConfig = nil
```

Comment:
why are you niling this out for all tables?

Reply:
I think this is sound since we aren't trying to write any sessions, leases, or sql_instances. So the fact that the table is or is not mr shouldn't matter to any of the data we are restoring.